buttons/main.scss

```diff
@@ -1,15 +1,8 @@
 @include nUiCriticalStart('buttons');
 
-$_n-ui-buttons-applied: false !default;
-$_n-ui-buttons-is-silent: nUiHas('buttons') == false or $_n-ui-buttons-applied == true;
-$o-buttons-is-silent: $_n-ui-buttons-is-silent;
+$o-buttons-is-silent: false !default;
```

Comment:
All Origami components do this for us: https://github.com/Financial-Times/o-typography/blob/master/main.scss#L141